src/api/swagger.yaml

```diff
@@ -2776,6 +2776,11 @@ components:
         monetaryZoneId:
           type: string
           description: The monetary Zone
+        fxpCurrencies:
+          type: array
+          description: List of FX currencies
+          items:
+            type: string
```

Comment:
I think this field can be generalised like 'supportedCurrencies'. Normal DFSPs can also support multiple currencies.

Reply:
This is supposed to be used specifically for FX currencies when adding FXP participant. As `supportedCurrencies` are something that is used in the ALS result, this property is to distinguish from that usage case. So we can have additional property for that case.